### PR TITLE
Update test tools and readme

### LINKED ISSUE #6: Tweaks
* Use latest AVA version
* Use XO
* Update `readme` to ES2015 (`var` -> `const`)

### PR DESCRIPTION
**Pull request by distinctblur**
Attempt to resolve #6. Test now fails with: 

```
$ npm test

> sparkly@2.0.1 test sparkly
> xo && node test.js


sparkly/cli.js
  30:9  error  Parsing error: Illegal return statement

✖ 1 problem (1 error, 0 warnings)
npm ERR! Test failed.  See above for more details.
```

This appears to be related to https://github.com/sindresorhus/xo/issues/51 and https://github.com/eslint/eslint/issues/1158? Should we wait that out or is there a workaround?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "minimist": "^1.1.0"
   },
   "devDependencies": {
-    "ava": "0.0.4"
+    "ava": "^0.7.0",
```
Comment (SamVerschueren) [inline package.json]: Use `*` as version

File: package.json [hunk 2]
```diff
@@ -41,6 +41,7 @@
...
+    "xo": "^0.11.2"
```
Comment (SamVerschueren) [inline package.json]: Use `*` as version

File: test.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ test('creates graph', function (t) {
 	t.assert(sparkly([1, 50, 100]) === '▁▄█');
 	t.assert(sparkly([2, 4, 8]) === '▂▄█');
 	t.assert(sparkly([1, 2, 3, 4, 5]) === '▂▄▅▇█');
-    t.assert(sparkly([25, 45]) === '▅█');
+	t.assert(sparkly([25, 45]) === '▅█');
 	t.end();
```
Comment (SamVerschueren) [inline test.js]: Remove `t.end()`, not needed anymore in the latest AVA version.

File: test.js [hunk 4]
```diff
@@ -10,7 +10,7 @@ test('creates graph', function (t) {
 	t.assert(sparkly([1, 50, 100]) === '▁▄█');
 	t.assert(sparkly([2, 4, 8]) === '▂▄█');
 	t.assert(sparkly([1, 2, 3, 4, 5]) === '▂▄▅▇█');
-    t.assert(sparkly([25, 45]) === '▅█');
+	t.assert(sparkly([25, 45]) === '▅█');
```
Comment (SamVerschueren) [inline test.js]: Replace `t.assert` with `t.is`

```js
t.is(sparkly([25, 45]), '▅█');
```

File: package.json [hunk 5]
```diff
@@ -14,7 +14,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "node test.js"
+    "test": "xo && node test.js"
```
Comment (sindresorhus) [inline package.json]: `"test": "xo && ava"`

**Comment by sindresorhus** [comment]
And give the pull request a proper title.

**Comment by SamVerschueren** [comment]
:arrow_up: .travis.yml to use `stable` instead of `iojs`.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request closed without merging**